lib/Dialect/SPIRV/Serialization/Deserializer.cpp

```diff
@@ -132,6 +132,9 @@ class Deserializer {
   /// Gets the constant's attribute and type associated with the given <id>.
   Optional<std::pair<Attribute, Type>> getConstant(uint32_t id);
 
+  /// Gets the constants's integer attribute with the given <id>.
```

Comment:
/// ... Returns a null IntegerAttr if the given <id> is not registered or does not correspond to an integer constant.